js/controller.js

```diff
@@ -262,8 +270,40 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
+    setIPadVideoPosition: function() {
```

Comment:
- weird naming as well. I suggest setVideoCentering?
- Put a comment explaining this method.